starter-code/scripts/article.js

```diff
@@ -30,7 +31,11 @@ Article.prototype.toHtml = function() {
     3. article title,
     4. article body, and
     5. publication date. */
-  $newArticle.html(new Article('Kevin Bacon'));
+  $newArticle.find('a').html(this.author);
+  $newArticle.find('a').attr('href',this.authorUrl);
+  $newArticle.find('h1').html(this.title);
+  $newArticle.find('section').html(this.body);
+
```

Comment:
You are forgetting publication date. While the template has a base publication date on it of 2000-01-01 it needs to be updated with the article's own published on  date.